core/templates/domain/topic/topic-creation-backend-api.service.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to notify about creation of topic and obtain
+ * topic_id.
+ */
+
+export interface ITopicCreationBackend {
+  'abbreviated_name': string,
+  'name': string
```

Comment:
Remove the single quotes in properties that doesn't have `_`

Reply:
ok will remove the quotes